lr-cli/cortx_setup/commands/common_utils.py

```diff
@@ -109,6 +109,12 @@ def get_cortx_states():
     """
     return get_pillar_data('execution/cortx_components')
 
+def get_reset_states():
+    """
+    Get cortx states iopath controlpath and system states
```

Comment:
Update description:
`Get reset states system controlpath and iopath states`

Reply:
Done